IdentityCore/src/webview/embeddedWebview/challangeHandlers/ios/MSIDCertAuthHandler.m

```diff
@@ -71,19 +71,15 @@ + (void)setCustomActivities:(NSArray<UIActivity *> *)activities
 + (BOOL)completeCertAuthChallenge:(NSURL *)endUrl
 {
     MSID_LOG_WITH_CTX_PII(MSIDLogLevelInfo, nil, @"Complete cert auth challenge with end URL: %@", [endUrl msidPIINullifiedURL]);
-    
-    if (s_certAuthInProgress)
-    {
-        s_certAuthInProgress = NO;
-        return [s_systemWebViewController handleURLResponse:endUrl];
-    }
-    
-    return NO;
+    return [s_systemWebViewController handleURLResponse:endUrl];
```

Comment:
should we still check that cert auth is in progress here?

Reply:
Yes I think that makes sense. Addressed.